### PR TITLE
CRS management

### PR DESCRIPTION
**Pull request by metzm**
first steps to use WKT as main CRS definition and leave datum transformation parameters to PROJ: store WKT and spatail reference id (authority code) if available.
Test command with CRS authority name "IGNF" and code "WALL78UTM1S.WALLIS96":
`g.proj -w srid=IGNF:WALL78UTM1S.WALLIS96` 
See also issue #961

### CODE REVIEW

**Comment by metzm** [comment]
More explanations to this PR:
libgis functions are added to read and write SRID and WKT definitions for a GRASS location.
SRID is a spatial reference ID composed of authority name and code, e.g. EPSG:4326
WKT is the latest WKT2 version if compiled against GDAL3+, otherwise WKT1.
The reasoning is that the GRASS native format to store projection information and in particular projection information needed for reprojection to another CRS is superseded by the mechanisms provided by PROJ6+. While WKT2 is a more suitable format to store projection information of a GRASS location, the preferred way is to have authority name and code, where the authority name is not restricted to EPSG.
The next step is to modify the GRASS reprojection system which can be done after this PR has been merged.

File: lib/gis/proj3.c [hunk 1]
```diff
@@ -11,6 +11,9 @@
   \author Original author CERL
  */
 
+/* TODO: the G_database_*() functions should be renamed to G_location_*()
```
Comment (marisn) [inline lib/gis/proj3.c:14]: This should happen before G8 or will have to wait till G9.

Comment (metzm) [reply lib/gis/proj3.c:14]: Renaming the `G_database_*()` functions to `G_location_*()` should be a separate PR, this is only cosmetics to clarify function names.
This PR adds new functionality and needs to be merged before the GRASS reprojection system can be changed to make use of this new functionality.

Comment (marisn) [reply lib/gis/proj3.c:14]: I agree. Just a heads-up to be clear on deadlines.

**Pull request merged**